web/views/pages/index.ejs

```diff
@@ -159,7 +163,7 @@
                                     <div id="country-emission-rect" class="country-col-box emission-rect emission-rect-overview">
                                         <div><span class="country-emission-intensity"></span>g</div>
                                     </div>
-                                    <div class="country-col-subtext">(CO2eq/kWh)</div>
+                                    <div class="country-col-subtext">(CO<sub>2</sub>eq/kWh)</div>
```

Comment:
CO2 appears a lot of places (including in specific translations.
Either we replace all CO2 by CO<sub>2</sub>, or we leave it like this.
I think it's better to leave it like it was before to be honest.